packages/reports/addon/components/navi-date-range-picker.js

```diff
@@ -44,7 +44,23 @@ export default Ember.Component.extend({
   /**
    * @property {Object} customRange - custom range interval, or default interval
    */
-  customRange: computed.or('interval', 'defaultInterval'),
+  customRange: computed.or('intervalInstance', 'defaultInterval'),
+
+  /**
+   * @property {Object} intervalInstance - Interval Object that is either passed in as an attribute or built from a string
+   */
+  intervalInstance: computed('interval', function() {
+    let interval = get(this, 'interval');
```

Comment:
Can we normalize to a interval class and change from string when we pass the prop down?